### PR TITLE
cash-1282 inital exp setup work

### PR DESCRIPTION
**Pull request by BoulderBrains**

### CODE REVIEW

File: src/components/Checkout/DonationItem.vue [hunk 1]
```diff
@@ -9,15 +9,25 @@
...
-					@click.prevent="triggerDefaultLightbox"
-					v-kv-track-event="['basket', 'Donation Info Lightbox', 'Open Lightbox']"
+			<div v-if="hasLoans">
+				<div
+					v-if="specificDonationUseTextExperiment"
+					class="donation-tagline small-text"
 				>
-					{{ donationDetailsLink }}
-				</a>
+					{{ specificDonationTagLine }}
```
Comment (BoulderBrains) [inline src/components/Checkout/DonationItem.vue]: Could also just hardcode the experiment text here if that would be prefered?

Comment (emuvente) [reply src/components/Checkout/DonationItem.vue]: Yeah since it's just static text go ahead and hardcode it here

Comment (BoulderBrains) [reply src/components/Checkout/DonationItem.vue]: ✔️

**Comment by BoulderBrains** [comment]
@mcstover Gonna move forward and merge this in to master for testing in dev. Please let me know if you have any issues with the code and I'll make the corrections.

**Pull request merged**